tracer/test/Datadog.Trace.Tests/SpanTests.cs

```diff
@@ -160,5 +160,114 @@ public void TopLevelSpans()
                 scope.Span.Should().Match<Span>(s => s.IsTopLevel == expectedResult);
             }
         }
+
+        [Fact]
+        public void SpanIds()
```

Comment:
I think each block should be a different unit test:
- Easier to debug
- Easier to read/maintain
- Easier to get specific error

Reply:
Done.

✔️ 

I noticed you blocked the PR. Is there any specific reason?

Reply:
oops, wrong button :)